util-hadoop/pom.xml

```diff
@@ -72,6 +72,11 @@
       <groupId>com.google.api-client</groupId>
       <artifactId>google-api-client-jackson2</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Declare this as managed dependency in main `pom.xml` and add property to configure version (see other dependencies)